### ISSUE TITLE
React 17 Support in peer dependencies, failed to install dependencies via npm v7

### ISSUE THREAD

**Issue by YF1999** [xobotyi/react-scrollbars-custom#184]
I have a project using react v17.0.2 and react-scrollbars-custom v4.0.25, and I am trying to use npm v7 now to manage dependencies.

npm v7 has stricter limit to peer dependencies, error occurs when installing:

```shell
❯ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @ysoftwarerepo/template-electron-cra@0.6.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0" from react-scrollbars-custom@4.0.25
npm ERR! node_modules/react-scrollbars-custom
npm ERR!   react-scrollbars-custom@"^4.0.25" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Also I checked #176 , if react-scrollbars-custom v4 is compatible with react v17, please change the peer dependencies.

**A little about versions:**  
- _OS_: Windows 10 Pro 20H2 19042.1083
- _Browser (vendor and version)_: Chromium 91.0.4482.124 (electron 13.1.7)
- _React_: 17.0.2
- _`react-scrollbars-custom`_: 4.0.25
- _Did this worked in the previous package version?_

**Comment by xobotyi** [comment]
I don't understand, peer dependency is `>= 16.0.0` where did I it found `^16`🤔

**Comment by YF1999** [comment]
Oh I see, the reason is that you have changed the react peer dependencies from `v16.0.0` to `>=16.0.0`, but did not publish a new version.

**Comment by xobotyi** [comment]
Fffuuuuu I forgot that here is no semantic-release configured for this project🙈🙈

**Comment by xobotyi** [comment]
So easy to get use to good thing😅

**Comment by YF1999** [comment]
Is there any plans about when the new release will be published?

**Comment by xobotyi** [comment]
@YF1999 just released 4.0.26
